aws/request/request_retry_test.go

```diff
@@ -0,0 +1,107 @@
+package request
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+)
+
+func r(t *testing.T, url string) *http.Request {
+	r, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		t.Fatalf("can't forge request: %v", err)
+	}
+	return r
+}
+
+func Test_shouldRetryCancel_nil(t *testing.T) {
+	shouldRetryCancel(nil)
+}
+
+func Test_shouldRetryCancel_timeout(t *testing.T) {
```

Comment:
Could you update these tests names to camel case like: `TestShouldRetryCancel`  having a `_<condition>` suffix is fine for specific sub cases. e.g. `TestShouldRetryCancel_timeout`

Reply:
Gotcha, done. This underscore happened when I generated a unit test using vscode.